src/plugins/data_source_management/public/components/data_source_menu/data_source_menu.tsx

```diff
@@ -108,6 +115,8 @@ export function DataSourceMenu<T>(props: DataSourceMenuProps<T>): ReactElement |
         return renderDataSourceView(componentConfig as DataSourceViewConfig);
       case DataSourceComponentType.DataSourceMultiSelectable:
         return renderDataSourceMultiSelectable(componentConfig as DataSourceMultiSelectableConfig);
+      case DataSourceComponentType.NoDataSource:
+        return renderNoDataSource(componentConfig as DataSourceBaseConfig);
```

Comment:
This is an internal component that we render when there is no data sources

Reply:
will remove